### DISCUSSION TITLE
Detect when generation goes crazy and regenerate immediately

### DISCUSSION THREAD

**Discussion by ben-barnard** [neonbjb/tortoise-tts#125] [General]
I was playing with the code and having it read some poetry when I would run into the random line where the generation would go crazy - and I understand that this is normal and you can regenerate once it's identified. But has anybody come up with a way to flag these occurrences so we have a shot at regenerating in the moment or immediately afterwards without interaction?

**Comment by I-Have-No-Idea-What-IAmDoing** [comment]
#91 Might be related issue?

**Reply by ben-barnard** [reply]
Yeah you're right. Going to look at those suggestions first. Thanks!
